### ISSUE TITLE
Require --dev or Publish a Non-Dev Version

### ISSUE THREAD

**Issue by grant** [GoogleCloudPlatform/functions-framework-php#10]
Currently the package requires the `--dev` install command for the README to work.

We should either publish this package as a non-dev version or make sure the quickstart works by adding the `--dev` flag`.

```
composer require --dev google/cloud-functions-framework
```

**Comment by bshaffer** [comment]
`--dev` in composer means the dependencies are only used when in development and should not be installed in production. Our example `Dockerfile` explicitly installs composer dependencies with the `--no-dev` flag, so supporting `--dev` for this library does not make sense, as it is not a dev dependency.

Currently, the package is installable by using `dev-master`, as there is no tagged release. Once this library is considered stable enough for a tag, we will tag it with `0.1`.

**Comment by bshaffer** [comment]
See `examples/hello/composer.json`:

```
{
    "require": {
        "google/cloud-functions-framework": "dev-master"
    }
}
``` 

this will work until we tag `0.1`

**Comment by grant** [comment]
We can't have instructions that don't work in the public README.
Perhaps we can alter the instructions to use the `composer` file above we suggested.

**Comment by bshaffer** [comment]
This has been fixed now that `0.1` is tagged
